acoustid/data/fingerprint.py

```
@@ -189,7 +188,9 @@ def _search_via_fpstore(
         )
         if self.timeout:
             timeout_ms = int(self.timeout * 1000)
-            self.db.execute("SET LOCAL statement_timeout TO {}".format(timeout_ms))
+            self.db.execute(
+                text("SET LOCAL statement_timeout TO {}".format(timeout_ms))
```

Ruff (0.8.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)